pkg/processor/batchprocessor.go

```diff
@@ -68,7 +61,7 @@ func (s *segmentsBatch) send(batch []*string) {
 }
 
 func (s *segmentsBatch) poll() {
-	failedAttempt := 0
+	//failedAttempt := 0
```

Comment:
Let's delete this instead of commenting it out.

Reply:
Thanks for catching that!